### PR TITLE
Add Specifically Clementines

### PR DESCRIPTION
**Pull request by davideshay**
Add grocery-shopping and recipe management app "Specifically Clementines" to the self-hosted data

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Comment by KieranRobson** [comment]
> https://github.com/awesome-selfhosted/awesome-selfhosted/pull/3962

[Original release](https://github.com/davideshay/groceries/releases/tag/v0.3.0) is now 4 months old so this could qualify

File: software/specifically-clementines.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: "Specifically Clementines"
+website_url: "https://davideshay.github.io/groceries/"
+source_code_url: "https://github.com/davideshay/groceries"
+description: "Grocery shopping app previously called 'groceries'. Reliable sync with multiple users/devices (web/mobile/Android/iOS). Available on Google Play Store. Includes recipes and integration with Tandoor."
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - Javascript
+  - Nodejs
```
Comment (nodiscc) [inline software/specifically-clementines.yml:10]:
```suggestion
platforms:
  - Docker
```

Docker is the only supported installation method (https://davideshay.github.io/groceries/installation/installation/)

> Values for platform are the main server-side requirements for the software

File: software/specifically-clementines.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+name: "Specifically Clementines"
+website_url: "https://davideshay.github.io/groceries/"
+source_code_url: "https://github.com/davideshay/groceries"
+description: "Grocery shopping app previously called 'groceries'. Reliable sync with multiple users/devices (web/mobile/Android/iOS). Available on Google Play Store. Includes recipes and integration with Tandoor."
```
Comment (nodiscc) [inline software/specifically-clementines.yml:4]:
```suggestion
description: "Grocery shopping app (previously Groceries), providing reliable sync with multiple users/devices (web/Android/iOS), recipes and integration with Tandoor."
```

File: software/specifically-clementines.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - Javascript
+  - Nodejs
+tags:
+  - Recipe Management
+  - Task Management & To-do Lists
+depends_3rdparty: no
```
Comment (nodiscc) [inline software/specifically-clementines.yml:14]:
```suggestion
```

`no` is already the default value, let's remove this

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by davideshay** [comment]
I think I updated as requested, let me know if there are other buttons I failed to press.

**Review by nodiscc — APPROVED** [review]

**Pull request merged**